### PR TITLE
Fix for llm examples with openvino-nightly

### PR DESCRIPTION
**Pull request by ljaljushkin**
### Changes

Explicit enabling of dynamic quantization of activations and updating metrics in the weight compression examples. 

It's possible to adapt compression examples for new openvino-nightly by explicitly turning off the dynamic quantization of activations via 

```
OVModelForCausalLM.from_pretrained(OUTPUT_DIR, ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "0"})
```
The successful 45th run of `test examples` is proving that. 
But examples with dynamic quantization should make validation faster, that's why this PR enables it and updates the metrics. 

### Reason for changes

OpenVINO 2023.2 has a dynamic quantization of activation enabled by default. (https://github.com/openvinotoolkit/openvino/pull/25054) and that's affects accuracy/metrics in the weight compression examples/tests, since before it was disabled by default.

### Related tickets

[CVS-134931](https://jira.devtools.intel.com/browse/CVS-134931)
[CVS-145701](https://jira.devtools.intel.com/browse/CVS-145701)

### Tests
`tests/cross_fw/examples/test_examples.py -k llm`

run of test examples:
- [x]  45 - with openvino-nightly

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]

**Comment by ljaljushkin** [comment]
@openvinotoolkit/nncf-maintainers please don't merge, the CI with current openvino is not passing for some reason. Will continue debugging.

File: examples/llm_compression/openvino/tiny_llama/main.py
Comment (l-bat) [inline examples/llm_compression/openvino/tiny_llama/main.py:70]: Do we also need to set "KV_CACHE_PRECISION": "u8"`?

Comment (ljaljushkin) [reply examples/llm_compression/openvino/tiny_llama/main.py:70]: in my experiments it doesn't change anything

**Comment by ljaljushkin** [comment]
> @openvinotoolkit/nncf-maintainers please DO NOT MERGE, the CI with current openvino is not passing for some reason. Will continue debugging.

@alexsu52 @l-bat ready for review and merge. I've just left the first commit with disabling dynamic quantization
build 45 passed with it.

**Review by l-bat — APPROVED** [review]

**Pull request merged**
